### DISCUSSION TITLE
Problem using start script on the HPC

### DISCUSSION THREAD

**Discussion by mirizarry-ortiz** [hytest-org/hytest#398] [Computing Environment]
Hi! I followed the instructions here:
https://hytest-org.github.io/hytest/environment_set_up/StartScript.html
to start the jupyter server on denali and I am having some trouble.
First, I do not get a 127.0.0.1 URL when I run jupyter-start.sh and instead I get this error:
![image](https://github.com/hytest-org/hytest/assets/69687785/31f97117-27ca-4ebd-b5a7-28718225bd46)

My local terminal says this:
![image](https://github.com/hytest-org/hytest/assets/69687785/5d21eb8d-8c0f-4a32-827d-9d368ae8b76d)

I was able to start a jupyter server via an older version of the start_jupyter script but it is quite different from the current script and I cannot get the Dask functionality to work on my notebooks when using the old script.

Can anyone help? Any ideas of what might be going on? The only modifications I made on jupyter-start.sh were these:
![image](https://github.com/hytest-org/hytest/assets/69687785/70cbbdd9-7852-4d44-97ae-d5397daae03a)
I have been trying to get this working for a few weeks now but I always get the same error message for srun.

I've been trying to understand the error message and all I've come up is that it happens when one is trying to run a job within a job. I wonder if it has to do with the fact that we are doing two calls to srun (srun --pty bash within the salloc command) and then at the end of jupyter-start.sh we call srun again:
srun jupyter lab --ip '*' --no-browser --port $JPORT --notebook-dir $PWD

**Comment by mirizarry-ortiz** [comment]
If I remove the srun --pty bash from the salloc command, there is more progress, but I still get the channel 4: open failed errors on the local terminal:
Denali terminal:
![image](https://github.com/hytest-org/hytest/assets/69687785/3aba860f-4a98-4e8f-b5f2-b476df27400c)

Local terminal:
![image](https://github.com/hytest-org/hytest/assets/69687785/d44f2827-0cfa-40d4-bbcd-1d403ee93559)

**Comment by amsnyder** [comment]
Hi @mirizarry-ortiz - thank you for raising this and doing some investigative work! You were right that the issue was happening because of the double `srun` commands. The `srun` inside of our start script needed to be removed. I fixed this with #399. Could you pull the latest copy of the repository onto Denali and try again? I think it will work for you now.

**Comment by amsnyder** [comment]
Discussion transferred to issue #400

**Reply by mirizarry-ortiz** [reply]
Yup! It's working now! Thanks! Now I just go to get the Dask cluster working! Thanks so much!
